event_sink_clickhouse/management/commands/dump_data_to_clickhouse.py

```diff
@@ -147,6 +149,18 @@ def add_arguments(self, parser):
             type=int,
             help="maximum number of objects to dump, cannot be used with '--ids' or '--force'",
         )
+        parser.add_argument(
+            "--batch_size",
+            type=int,
+            default=1000,
```

Comment:
Was 10,000 a good number for your test? Not sure if this should be updated.

Reply:
Yes, a great number with low resource consumption and high throughput